bus-mapping/Cargo.toml

```diff
@@ -13,7 +13,8 @@ lazy_static = "1.4"
 serde_json = "1.0.66"
 hex = "0.4"
 geth-utils = { path = "../geth-utils" }
-web3 = {version = "0.17", default-features = false}
+# web3 = {version = "0.17", default-features = false}
+web3 = { git = "https://github.com/tomusdrw/rust-web3.git", rev = "1f807bb7c288a1d2668c691839381786065b2eac" }
```

Comment:
The releasing cycle of `rust-web3` is quite fast. Is `Proof` the only thing we need from the new version?

Reply:
Yes, just need new version.

Reply:
Currently we're using `web3` for the types and `ethers` for the json-rpc; but `ethers` also provides the ethereum types.  In https://github.com/appliedzkp/zkevm-circuits/pull/201 I migrated the types from `web3` to `ethers` to remove the dependency to `web3`.  I see that `ethers` has the Proof type already in a release: https://docs.rs/ethers-core/0.6.1/ethers_core/types/struct.StorageProof.html

So in case #201 is merged first, I think it would be nice if you can update this PR to use the `StorageProof` type from `ethers` :)

Reply:
Ohh I completely forgot. Sorry @ed255  my bad. 

Ye, in that case I completely agree with Edu's point here. Let's wait till #201 gets merged and then use the new types! :)

Reply:
sounds good!:)